course_content/notebooks/iris_intro.ipynb

```diff
@@ -796,7 +757,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Because Iris cells represent both point and bound, cell comparison can sometimes be counter-intuitive:"
+    "### Time Constraints\n",
+    "\n",
+    "It is common to want to build a constraint for time. This can be achieved by comparing cells containing datetimes. The functionality can be enabled globally within the session (and will be enabled by default in v2.0, the next release of Iris) with:"
```

Comment:
This sentence is fine, I just think it might be something that we should take a few minutes over while teaching.

I know the state of affairs with this iris.FUTURE toggle, and even I had to read this sentence two or three times to make sure that it was in line with what I knew.  It might just need a bit of clarification for those who are unfamiliar with it.

Reply:
Yeah, both times I've taught it I've used this cell as a springboard to actually explore this a little. (Note I've also entirely removed one of the options for constraining on time from the notebook, both for the sake of clarity and length.)